src/Uno.Extensions.Http.Refit/ServiceCollectionExtensions.cs

```diff
@@ -76,4 +79,111 @@ public static IServiceCollection AddRefitClientWithEndpoint<TInterface, TEndpoin
 			}),
 			configure: configure);
 	}
+
+	/// <summary>
+	/// Registers a Kiota client with the specified <paramref name="name"/>.
+	/// </summary>
+	/// <typeparam name="TClient">The Kiota client type to register</typeparam>
+	/// <param name="services">The services collection to register the client with</param>
+	/// <param name="context">The host builder context</param>
+	/// <param name="options">[optional] Endpoint information (loaded from appsettings if not specified)</param>
+	/// <param name="name">[optional] The name for locating endpoint information in appsettings</param>
+	/// <param name="configure">[optional] Callback for configuring the endpoint</param>
+	/// <returns>The updated service collection</returns>
+	public static IServiceCollection AddKiotaClient<TClient>(
```

Comment:
Was it intentional to add this to the refit library? Seems like the two services, refit and kiota, have too much overlap to be used together?

Reply:
@nickrandolph just trying to test all this with a local build, so moved everything to refit to test it with my Chefs server project, all of it will be removed from the refit project after, only keeping the kiota project.